index.html

```diff
@@ -0,0 +1,350 @@
+<!doctype html>
+<html class="no-js" lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Avengers</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+
+    <link rel="stylesheet" href="/css/reset.css">
+    <link rel="stylesheet" href="/css/style.css">
+  </head>
+  <body>
+    <nav class="menu">
+      <a href="#home">Home.</a>
+      <a href="#services">Services.</a>
+      <a href="#portfolio">Portfolio.</a>
+      <a href="#about">About.</a>
+      <a href="#team">Team.</a>
+      <a href="#contact">Contact.</a>
+    </nav>
+    <header id="home" class="header-content">
+      <img class="logo circle" width="21" height="32" src="/img/tear-drop-two.svg" alt="Logo for Avengers" title="Avengers logo">
```

Comment:
I really like how you solved this -- a lot of people would've dropped both the background circle _and_ the drop in to the loaded SVG, but by using CSS for that circle, you open up the ability to theme the logo. Well done.